rpc/core/net.go

```diff
@@ -47,9 +48,13 @@ func NetInfo() (*ctypes.ResultNetInfo, error) {
 			ConnectionStatus: peer.Status(),
 		})
 	}
+	// TODO: Should we include PersistentPeers and Seeds in here?
```

Comment:
yeah, let's do this

Reply:
opened an issue: https://github.com/tendermint/tendermint/issues/1526